src/js/common/schemaform/ObjectField.jsx

```diff
@@ -117,6 +118,15 @@ class ObjectField extends React.Component {
     return _.values(groupedProperties);
   }
 
+  showPrefill(uiOptions, formContext) {
+    const isProduction = process.env.__BUILDTYPE__ === 'production';
```

Comment:
You should be able to leave off the `process.env` part.

Reply:
Thanks @jbalboni, removed.